mozillians.tf

```diff
@@ -0,0 +1,98 @@
+    description = "mozillians slave production SG"
+    vpc_id      = "${aws_vpc.apps-production-vpc.id}"
+}
+
+resource "aws_security_group_rule" "mozillians-slave-ec2-sg-allowallfrommaster" {
+    type                     = "ingress"
+    from_port                = 0
+    to_port                  = 0
+    protocol                 = "-1"
+    source_security_group_id = "${module.mesos-cluster-production.mesos-cluster-master-sg-id}"
+
+    security_group_id        = "${aws_security_group.mozillians-slave-ec2-sg.id}"
+}
+
+resource "aws_security_group_rule" "mozillians-slave-ec2-sg-allowall" {
+    type              = "egress"
+    from_port         = 0
+    to_port           = 0
+    protocol          = "-1"
+    cidr_blocks       = ["0.0.0.0/0"]
+
+    security_group_id = "${aws_security_group.mozillians-slave-ec2-sg.id}"
+}
+
+resource "aws_security_group_rule" "mozillians-slave-ec2-sg-allowallfromshared" {
+    type              = "ingress"
+    from_port         = 0
+    to_port           = 0
+    protocol          = "-1"
+    cidr_blocks       = ["${aws_vpc.apps-shared-vpc.cidr_block}"]
+
+    security_group_id = "${aws_security_group.mozillians-slave-ec2-sg.id}"
+}
+
+resource "aws_launch_configuration" "mozillians-slave-ec2-lc" {
+  name_prefix                 = "mozillians-slave-production-lc"
+  image_id                    = "${lookup(var.aws_amis, var.aws_region)}"
+  instance_type               = "t2.medium"
+  key_name                    = "ansible"
+  security_groups             = ["${aws_security_group.mozillians-slave-ec2-sg.id}"]
+  associate_public_ip_address = true
+  root_block_device {
+    volume_size               = 100
+  }
+  lifecycle {
+    create_before_destroy     = true
+  }
+}
+
+resource "aws_autoscaling_group" "mozillians-slave-as" {
+    name                    = "mozillians-slave-production-as"
+    launch_configuration    = "${aws_launch_configuration.mozillians-slave-ec2-lc.id}"
+    availability_zones      = ["${split(",", lookup(var.aws_availibility_zones, var.aws_region))}"]
+    max_size                = "2"
+    desired_capacity        = "2"
+    min_size                = "2"
+    vpc_zone_identifier     = ["${aws_subnet.apps-production-1a.id}", "${aws_subnet.apps-production-1c.id}", "${aws_subnet.apps-production-1d.id}"]
+    tag {
+      key                   = "app"
+      value                 = "mozilliansslaveproduction"
+      propagate_at_launch   = true
+    }
+
+    tag {
+      key                   = "env"
+      value                 = "production"
+      propagate_at_launch   = true
+    }
+
+    tag {
+      key                   = "Name"
+      value                 = "mozilliansslave"
+      propagate_at_launch   = true
+    }
+
+    tag {
+      key                   = "project"
+      value                 = "mozillians"
+      propagate_at_launch   = true
+    }
+
+    lifecycle {
+      create_before_destroy = true
+    }
+}
+
+module "mozillians-staging" {
```

Comment:
Do we need to separate the staging nodes from prod? I think having mozillians.org nodes for all the envs should be fine.